### ISSUE TITLE
listUnorderedItem and other list styles do not work

### ISSUE THREAD

**Issue by jayshah123** [iamacup/react-native-markdown-display#48]
tried applying following:
```
const bodyStyles = StyleSheet.create({
  listUnorderedItem:{
    color: 'yellow'
  },
  listUnordered: {
    color: 'green'
  },
  listItem: {
    color: 'white'
  },
});
```
But it has no effect.
The list item text still shows in black.

Other styles like paragraph, text, heading1, etc work fine.

**Comment by jayshah123** [comment]
I think all the text within a list item picks up "paragraph" style as well.

**Comment by iamacup** [comment]
Nice catch - ill take a look

**Comment by jayshah123** [comment]
I used https://astexplorer.net/ to find out the parse tree, but it uses a slightly different parser.

**Comment by iamacup** [comment]
@jayshah123 this is fixed in 6.0.0
